comms/discovery/config/constants.go

```diff
@@ -15,7 +15,7 @@ var (
 		RateLimitTimeframeHours:             24,
 		RateLimitMaxNumMessages:             2000,
 		RateLimitMaxNumMessagesPerRecipient: 1000,
-		RateLimitMaxNumNewChats:             100,
+		RateLimitMaxNumNewChats:             100000,
```

Comment:
this is just the default value set in code. Does anyone know if we are using an actual config file in prod? If so, we'll need to update that so it doesn't eclipse this value.